src/app/frontend/common/components/allocatedresourceschart/component.js

```diff
@@ -25,6 +25,17 @@ export class AllocatedResourcesChartController {
     /** @private {!angular.JQLite} */
     this.element_ = $element;
 
+    /**
+     * Data that fills a single pie chart
+     * @export {Object}
+     */
+    this.data;
+
+    /**
+     * Colors for single ring pie chart
+     * @export {!Array.<string>}
+     */
+    this.colorpalette;
```

Comment:
It looks like making this camelCase actually causes things to break (array is not passed through from `<kd-allocated-resources-chart colorPalette=$ctrl.colorPalette>`. Any ideas?